src/navigation/onNavigationStateChange.js

```diff
@@ -35,7 +35,7 @@ export function onHandleStatusBar(currentState, prevState) {
     StatusBarHelper.setLightContent();
     return;
   }
-  const isFromWalletScreen = Navigation.getActiveRoute().params
+  const isFromWalletScreen = Navigation.getActiveRoute()?.params
```

Comment:
This fix was needed because there is no active route in the design system playground.